otter/cloud_client.py

```diff
@@ -1051,3 +1051,62 @@ def publish_to_cloudfeeds(event, log=None):
             'content-type': ['application/vnd.rackspace.atom+json']},
         data=event, log=log, success_pred=has_code(201),
         json_response=False)
+
+
+# ----- Cloud orchestration requests -----
+def list_stacks_all(parameters=None):
+    """
+    List Heat stacks.
+
+    :ivar dict parameters: Query parameters to include.
+
+    Succeed on 200.
+
+    :return: a `tuple` of (:obj:`twisted.web.client.Response`, JSON `dict`)
+    """
+
+    # TODO Actually implement batch handling
```

Comment:
Is this going to be necessary before we deploy this feature? I assume if a user has over N(=20? 100? 500?) stacks then this will only get the first batch?